app/controllers/application_controller.rb

```diff
@@ -6,10 +6,14 @@ class ApplicationController < ActionController::Base
   rescue_from Errors::UnauthorizedError,  with: :render_not_found
   rescue_from ::Groups::NotFoundError,    with: :render_not_found
 
-  before_action :rails_rumble_context
+  before_action :rails_rumble_context, unless: :admin_controller?
```

Comment:
We can get rid of it but I think it should be done in a different PR.
So for now, I'm just disabling it for admin :grimacing: